src/ui/components/PropertyRow/PropertyRow.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+import { Stack } from "@deskpro/deskpro-ui";
+
+import { ReactElement } from "react";
+
+const Divider = styled.div`
+  display: inline-block;
+  min-width: 1px;
+  height: 100%;
+  background-color: ${({ theme }) => theme.colors.grey20};
+  margin: 0px 6px 0px 0px;
+`;
+
+export const PropertyRow = ({ children }: { children: ReactElement[] }) => {
+  return (
+    <Stack style={{ position: "relative", width: "100%" }}>
```

Comment:
might want to pass `gap` to this from `PropertyRow` props as it'd be useful to adjust this